CLAUDE.md

```
@@ -0,0 +1,30 @@
+Always use mise tasks for running tests, linting, formatting. If there is not a command already present in .mise.toml, suggest creating one.
+
+## Code Style Guidelines
+
+Unless the code is complex, never use comments in the code. The code should speak for itself.
+
+## GitHub Issue Creation
+
+When asked to create a new GitHub issue:
+
+1. Follow the structure in template files:
+  - `.github/ISSUE_TEMPLATE/BUG.md.md` for bugs
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)